bokehjs/src/coffee/models/tickers/composite_ticker.coffee

```diff
@@ -39,7 +39,7 @@ export class CompositeTicker extends ContinuousTicker
     # this can happen if the data isn't loaded yet, we just default to
     # the first scale
     best_index = argmin(errors)
-    if best_index == Infinity
+    if best_index is undefined
       return @tickers[0]
```

Comment:
I would prefer not to depend on this corner case in `argmin()` and check for empty `errors` and compute argmin only if `errors` is non-empty.